scout/build/case.py

```diff
@@ -31,7 +28,7 @@ def build_phenotype(phenotype_id, adapter):
     if phenotype:
         phenotype_obj["phenotype_id"] = phenotype["hpo_id"]
         phenotype_obj["feature"] = phenotype["description"]
-    return phenotype
+    return phenotype_obj
```

Comment:
According to github, this has been riding with us for at least 7 years. It will be interesting/scary to see if there are any downstream issues with correcting it. Almost as scary if there are none. X)